apps/web/src/ui/pool/V3PositionView.tsx

```diff
@@ -668,18 +674,14 @@ const Component: FC<{ chainId: string; address: string; position: string }> = ({
                 </CardContent>
                 <TabsContent value="active">
                   <DistributionDataTable
-                    isLoading={rewardsLoading}
-                    data={currentAngleRewardsPool?.distributionData.filter(
-                      (el) => el.isLive,
-                    )}
+                    isLoading={isCampaignsLoading}
+                    data={campaignsData?.filter((el) => el.isLive)}
                   />
                 </TabsContent>
                 <TabsContent value="inactive">
                   <DistributionDataTable
-                    isLoading={rewardsLoading}
-                    data={currentAngleRewardsPool?.distributionData.filter(
-                      (el) => !el.isLive,
-                    )}
+                    isLoading={isCampaignsLoading}
+                    data={campaignsData?.filter((el) => !el.isLive)}
```

Comment:
Unstable, might make the table rerender every time. We should probably finally look into enabling the React compiler.